### PR TITLE
Add Grafana to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**

### CODE REVIEW

**Review by berksmbl — APPROVED** [review]
Everything seems fine.

File: templates/grafana/index.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import {
+  Output,
+  Services
+} from "~templates-utils";
```
Comment (Ponkhy) [inline templates/grafana/index.ts:4]:
```suggestion
import { Output, Services } from "~templates-utils";
```

File: templates/grafana/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: Grafana
+description: The open and composable observability and data visualization platform
+instructions: Login with admin:admin
+changeLog:
+  - date: 2023-3-15
+    description: first release
+links:
```
Comment (Ponkhy) [inline templates/grafana/meta.yaml:7]: You could also add the Demo page `https://play.grafana.org/`.

File: templates/grafana/index.ts (hunk 1 above)
Comment (berksmbl) [reply templates/grafana/index.ts:4]: I totally agree, I recommend using the [prettier](https://prettier.io/) app.

**Comment by Supernova3339** [comment]
Andrei runs it whenever PRs get merged, I don't touch it as it messes up my
flow

File: templates/grafana/meta.yaml (hunk 2 above)
Comment (Supernova3339) [reply templates/grafana/meta.yaml:7]: not implemented

**Review by Ponkhy — APPROVED** [review]
LGTM

**Pull request merged**